supervisor/host/configuration.py

```diff
@@ -46,6 +55,14 @@ class IpSetting:
     nameservers: list[IPv4Address | IPv6Address]
 
 
+@dataclass(slots=True)
+class Ip6Setting(IpSetting):
+    """Represent a user IPv6 setting."""
+
+    addr_gen_mode: InterfaceAddrGenMode = InterfaceAddrGenMode.DEFAULT_OR_EUI64
+    ip6_privacy: InterfaceIp6Privacy = InterfaceIp6Privacy.DISABLED
```

Comment:
With this, if IPv6 is not set we actually set explicit `addr-gen-mode` and `ip6-privacy` via D-Bus.

To make the system behave as it is currently, we should choose defaults which don't set those modes, so:

```suggestion
    addr_gen_mode: InterfaceAddrGenMode = InterfaceAddrGenMode.DEFAULT
    ip6_privacy: InterfaceIp6Privacy = InterfaceIp6Privacy.DEFAULT
```

With that change, the two modes don't get written via D-Bus, making sure that if things stay the way they are today if not explicitly set.

Reply:
But didn't you said it's dangerous to set addr_gen_mode to 3 ? In terms of backward compatibility?

Reply:
Hm, true, just didn't think that this slight misalignment between the two configs is acceptable. But I guess it ONLY applies fot those older OS's so it's most likely fine. In terms of clarity. 😉